### DISCUSSION TITLE
Support for new (not legacy) Projects?

### DISCUSSION THREAD

**Discussion by mauvehed** [octokit/octokit.rb#1603] [Q&A] [answered]
I'm working to migrate some old ruby code that uses octokit with what are now the legacy Projects on GitHub. What I'm not clear on is whether octokit supports the new Projects or not.

It seems, not.. but I am not really clear on that. If that is the case, what is the new recommend approach for interfacing with them?

**Comment by timrogers** [comment, marked as answer]
@mauvehed The new GitHub Projects aren't supported in Octokit, because they use the GraphQL API, and Octokit.rb is focused on the REST API. You can learn about how to work with Projects using the GraphQL API [here](https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/using-the-api-to-manage-projects). I think your best bet would be to use a generic GraphQL Ruby client and write your own queries.
